documentation/api/query/v4/nodes.markdown

```diff
@@ -15,8 +15,8 @@ Nodes can be queried by making an HTTP request to the `/nodes` endpoint.
 
 ## `GET /v4/nodes`
 
-This will return all nodes matching the given query. Deactivated nodes
-aren't included in the response.
+This will return all nodes matching the given query. Deactivated and expired
+nodes aren't included in the response.
```

Comment:
yeah, maybe. might be worth here as well since that goes into the mechanics a bit https://docs.puppetlabs.com/puppetdb/latest/maintain_and_tune.html

I feel like we additionally should have some kind of glossary page for this kind of stuff long term. Out of scope for this ticket though.